src/validation.cpp

```diff
@@ -1129,11 +1129,15 @@ static std::pair<CAmount, CAmount> GetBlockSubsidyHelper(int nPrevBits, int nPre
         dDiff = ConvertBitsToDouble(nPrevBits);
     }
 
-    if (fV20Active) {
-        // Starting from V20 Activation, subsidybase should be stable.
-        // Currently, nSubsidyBase calculate relies on difficulty.
-        // Once Platform is live, it must constantly get blocks difficulty in order to calculate platformReward.
-        // This can not be continued so we set the nSubsidyBase to a fixed value.
+    bool isDevnet = Params().NetworkIDString() == CBaseChainParams::DEVNET;
```

Comment:
<s>nit: `const bool isDevnet` - there's no intention to change this flag further</s>